worker/test/src/RTC/TestKeyFrameRequestManager.cpp

```diff
@@ -0,0 +1,64 @@
+#include "common.hpp"
+#include "DepLibUV.hpp"
+#include "catch.hpp"
+#include "RTC/KeyFrameRequestManager.hpp"
+#include "handles/Timer.hpp"
+
+using namespace RTC;
+
+SCENARIO("KeyFrameRequestManager", "[rtp][keyframe]")
+{
+	// Start LibUV.
+	DepLibUV::ClassInit();
+
+	class TestKeyFrameRequestManagerListener : public KeyFrameRequestManager::Listener
+	{
+	public:
+		void OnKeyFrameNeeded(uint32_t /*ssrc*/) override
+		{
+			this->onKeyFrameNeededTimesCalled++;
+		}
+
+	public:
+		size_t onKeyFrameNeededTimesCalled{ 0 };
+	};
+
+	static TestKeyFrameRequestManagerListener listener;
+
+	SECTION("KeyFrame requested once, not received on time")
```

Comment:
Can we reword "KeyFrame" to "key frame" everywhere in the text? This is how we name it in other places.